src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar_cell--wednesday calendar_cell--february">
```

Comment:
```suggestion
    <main class="calendar--wed calendar--28">
```
Модификаторы должны быть у класса calendar, а не у calendar_cell, и выглядеть вот таким образом

Reply:
Сейчас невозможно установить 29 дней в месяце, получается?

Reply:
![image](https://user-images.githubusercontent.com/89140295/141942482-a3e4a775-55ff-4c62-af5a-07c9572f46db.png)

Reply:
коли прописую
модифікатор calendar--february29
то в місяці 29 днів.
Якшо чесно шось не розумію шо не так?